src/lib/components/buy_product/BuyProduct.js

```diff
@@ -94,20 +101,34 @@ export default class BuyProduct extends Component {
       </Row>
     );
 
+    const backToProducts = (
+      <Column justify="start" align="center" style={{ width: '100%' }}>
+        <BackToProducts storeUrl="/" linkText="Back to Products" />
+      </Column>
+    );
+
+    const productVariants = (
+      <ProductVariants variants={otherVariants} store={store} />
+    );
+
     return (
-      <Row className={classes} justify="center" align="start">
-        {skuImageGallery}
-        <Column
-          className={`${CLASS_ROOT}__options`}
-          justify="start"
-          align="start"
-        >
-          {designTitle}
-          {optionSelector}
-          {designPrice}
-          {cartButton}
-        </Column>
-      </Row>
+      <div className={CLASS_ROOT}>
+        <Row className={classes} justify="center" align="start">
```

Comment:
Classes should move to the parent.
and internal rows should have classes something like this

className={`${CLASS_ROOT}__main`}
className={`${CLASS_ROOT}__other-products`}